src/index.js

```diff
@@ -11,6 +11,7 @@ import eventListenerOptions from './utils/eventListenerOptions';
 
 export type LazilyRenderProps = {
   className?: string;
+  appearance?: string;
```

Comment:
how about **tagName**? Is not to clear to me that appearance will change the resulting dom element